### PR TITLE
Style links and buttons

### LINKED ISSUE #83: Link is Gray but not Disabled
I noticed that the `visited links` are [gray](https://github.com/umts/st-pax/blob/master/app/assets/stylesheets/application.scss#L61). Whenever I see _anything_ on a page that is grayed out, I automatically think it's disabled. Compare the top two passengers' `view` and `delete` to the bottom one:
screen shot 2017-09-17 at 10 18 19 am

The reason I bring this up is because we have gray-on-gray:
screen shot 2017-09-17 at 10 28 28 am

And that same color indicates that a button is disabled:
screen shot 2017-09-17 at 10 21 41 am

My fix would be to remove styling on visited links. Or maybe remove disabled buttons from the user. Possibly both. Could be a personal problem?

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #83

### CODE REVIEW

**Comment by mboneil10** [comment]
Probably safe to remove disabling the checkbox logic?

File: app/views/passengers/_form.html.haml [hunk 1]
```diff
@@ -9,8 +9,9 @@
 = form_with model: @passenger, local: true do |form|
   .field
     = form.label :permanent
-    = form.check_box :permanent, disabled: @current_user.dispatcher?,
-      id: :passenger_permanent
+    - unless @current_user.dispatcher?
+      = form.check_box :permanent, disabled: @current_user.dispatcher?,
```
Comment (Anbranin) [inline app/views/passengers/_form.html.haml]: Why disable it if the current user is a dispatcher while in a block that won't run if the current user is a dispatcher?

File: app/views/passengers/_form.html.haml [hunk 2]
```diff
@@ -30,13 +31,15 @@
     = form.label :status
     = form.select :status, Passenger::STATUSES, { include_blank: true }, { id: :passenger_status }
   .field
-    = form.label :registered_with_disability_services
-    = form.check_box :registered_with_disability_services,
-      id: :pax_registered_with_DS, disabled: @passenger.permanent?
+    - unless @passenger.permanent?
+      = form.label :registered_with_disability_services
+      = form.check_box :registered_with_disability_services,
+        id: :pax_registered_with_DS, disabled: @passenger.permanent?
```
Comment (Anbranin) [inline app/views/passengers/_form.html.haml]: Same comment here.

File: app/views/passengers/_form.html.haml (hunk 1 above)
Comment (dfaulken) [reply app/views/passengers/_form.html.haml]: This is what @mboneil10 meant by

> Probably safe to remove disabling the checkbox logic?

File: app/views/passengers/_form.html.haml [hunk 3]
```diff
@@ -30,13 +31,15 @@
...
   .field
-    = form.label :has_brochure
-    = form.check_box :has_brochure,
-      id: :pax_has_brochure, disabled: @passenger.permanent?
+    - unless @passenger.permanent?
+      = form.label :has_brochure
+      = form.check_box :has_brochure,
+        id: :pax_has_brochure, disabled: @passenger.permanent?
```
Comment (Anbranin) [inline app/views/passengers/_form.html.haml]: I'm sensing a pattern here... am I wrong about this?

File: app/views/passengers/_form.html.haml (hunk 1 above)
Comment (Anbranin) [reply app/views/passengers/_form.html.haml]: I see now! Yes.

File: app/views/passengers/_form.html.haml (hunk 3 above)
Comment (mboneil10) [reply app/views/passengers/_form.html.haml]: Nope. It's up to you and @chauce if I should hide these or just keep it the way it was (disabling boxes).

Comment (Anbranin) [reply app/views/passengers/_form.html.haml]: My personal preference is that the box is just disabled but I feel not strongly at all about this. I can just imagine maybe _maybe_ some confusion on the part of the dispatcher at some point "hey, don't I have to check if this person is registered or has a brochure?" but it's really whatever.

Comment (mboneil10) [reply app/views/passengers/_form.html.haml]: Ok. That second part is what I was thinking about, too.

**Pull request merged**